packages/devtools_app/lib/src/memory/flutter/memory_heatmap.dart

```diff
@@ -153,6 +191,12 @@ class FlameChartState extends State<FlameChart> with AutoDisposeMixin {
   }
 }
 
+/// Key in callbacks map.
```

Comment:
Key in callbacks map for...? Add more descriptive comment. Same below for findNodeKey.

Reply:
Done.
```
/// Definitions of exposed callback methods stored in callback Map the key
/// is the function name (String) and the value a callback function signature.

/// matchNames callback key.
const matchNamesKey = 'matchNames';

/// matchNames callback signature.
typedef MatchNamesFunction = List<String> Function(String);
```